2021/day10/day10.java

```diff
@@ -0,0 +1,93 @@
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedList;
+
+public class day10 {
+    public static class Scores {
+        int totalSyntaxScore;
+        Long middleScore;
+
+        public Scores(int x, Long y) {
+            totalSyntaxScore = x;
+            middleScore = y;
+        }
+    }
+
+    public static ArrayList<ArrayList<Character>> readInput(String filename) {
+        ArrayList<ArrayList<Character>> input = new ArrayList<>();
+        Scanner sc = null;
+        try {
+            sc = new Scanner(new File(filename));
+            while (sc.hasNextLine()) {
+                ArrayList<Character> line = new ArrayList<>();
+                String s = sc.next();
+                for (int i = 0; i < s.length(); i++) {
+                    line.add(s.charAt(i));
+                }
+                input.add(line);
+            }
+        } catch (Exception e) {
+            System.out.println("Something went wrong" + e);
+        } finally {
+            sc.close();
+        }
+        return input;
+    }
+
+    public static HashMap<Character, Integer> initiateScores() {
+        HashMap<Character, Integer> map = new HashMap<>();
+        map.put(')', 3);
+        map.put(']', 57);
+        map.put('}', 1197);
+        map.put('>', 25137);
+        map.put('(', 1);
+        map.put('[', 2);
+        map.put('{', 3);
+        map.put('<', 4);
+        return map;
+    }
```

Comment:
Two things here:
1. I would separate these two, there should be a separate map for each score as they mean different things.
2. You can use a static map for this, see e.g. [this StackOverflow](https://stackoverflow.com/questions/507602/how-can-i-initialise-a-static-map). Personally I find that nicer than a function